disco_aws_automation/disco_cloudfront.py

```diff
@@ -0,0 +1,194 @@
+                    'ViewerProtocolPolicy': 'allow-all',
+                    'MinTTL': 86400,
+                    'AllowedMethods': {
+                        'Quantity': 2,
+                        'Items': [
+                        'HEAD',
+                        'GET',           
+                        ],
+                        'CachedMethods': {
+                            'Quantity': 2,
+                            'Items': [
+                                'HEAD',
+                                'GET'
+                            ]
+                        }
+                    },
+                    'SmoothStreaming': False,
+                    'DefaultTTL': 86400,
+                    'Compress': True
+                },
+                'CacheBehaviors': {
+                    'Quantity': 1,
+                    'Items': [
+                        {
+                            'PathPattern': '*',
+                            'TargetOriginId': full_origin_path,
+                            'ForwardedValues': {
+                                'QueryString': True,
+                                'Cookies': {
+                                    'Forward': 'none',
+                                    'WhitelistedNames': {
+                                        'Quantity': 0,
+                                    }
+                                },
+                                'Headers': {
+                                    'Quantity': 0,
+                                }
+                            },
+                            'TrustedSigners': {
+                                'Enabled': False,
+                                'Quantity': 0,
+                            },
+                            'ViewerProtocolPolicy': 'allow-all',
+                            'MinTTL': 86400,
+                            'AllowedMethods': {
+                                'Quantity': 2,
+                                'Items': [
+                                    'HEAD',
+                                    'GET',
+                                ],
+                                'CachedMethods': {
+                                    'Quantity': 2,
+                                    'Items': [
+                                        'HEAD',
+                                        'GET',
+                                    ]
+                                }
+                            },
+                            'SmoothStreaming': False,
+                            'DefaultTTL': 86400,
+                            'MaxTTL': 86400,
+                            'Compress': True
+                        },
+                    ]
+                },
+                'CustomErrorResponses': {
+                    'Quantity': 1,
+                    'Items': [
+                        {
+                            'ErrorCode': 404,
+                            'ResponsePagePath': '/Error.html',
```

Comment:
Hey Penny. Yeah. That's one of the assumptions I've made. Packaging an error page with the S3 seemed like the most straightforward way to go.
The way I envision it, all individual content modules can go in separate folders in the same bucket and they all reference the error page at the top level.